x-pack/plugins/lists/public/lists/hooks/use_delete_list.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useAsyncTask } from '../../common/hooks/use_async_task';
+import { DeleteListParams } from '../types';
+import { deleteList } from '../api';
+
+export type DeleteListTaskArgs = Omit<DeleteListParams, 'signal'>;
+
+const deleteListsTask = (
+  { signal }: AbortController,
+  args: DeleteListTaskArgs
+): ReturnType<typeof deleteList> => deleteList({ signal, ...args });
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Half the value of `useAsyncTask` is inferring arguments of the passed task, so this rule is ignored.

Reply:
Would it be worth adding this comment in the file itself for future reference?

Reply:
@yctercero that's a good idea, in the interest of build time I'll add that in the subsequent PR 😉